### DISCUSSION TITLE
Inline blue box?

### DISCUSSION THREAD

**Discussion by buhtz** [mkdocs/mkdocs#3720] [Q&A]
![image](https://github.com/mkdocs/mkdocs/assets/11861496/11a520c8-2334-4b9b-8f87-2a414f56e346)

See this "Settings->Auto Remove" with the blue box in the text? This screenshot is from a Sphinx generated docu.

Can I realize something like this with markdown in Mkdocs?

**Comment by pawamoy** [comment]
Use your browser de tools to inspect the CSS used for that blue box. Then add this CSS to your mkdocs.yml, and enable the `attr_list` Markdown extension. Then add the right CSS class to inline code?

```
See `Settings - Auto Remove`{ .blue-box }.
```

Something like that :)

**Reply by buhtz** [reply]
Mhm... that is a workaround/hack. I am able to do CSS but don't want to. I do use MkDocs to not doing things like this. The question is if there is a plubing/extension/fancything solving this?

**Reply by pawamoy** [reply]
Not that I know of. Feel free to check our [catalog](https://github.com/mkdocs/catalog/) :slightly_smiling_face: 
Do these blue boxes have an official denomination (a name commonly used by SSGs, to search in the catalog)?
